src/adaptor/mod.rs

```diff
@@ -0,0 +1,21 @@
+mod ini;
+
+use std::collections::HashMap;
+use std::io::{Read, Write};
+
+#[derive(Debug, Eq, PartialEq)]
+pub enum Config {
```

Comment:
I wonder if `Config` is a good name for this. I get it that files contain configuration, but at first sight, I had the impression that this was an object to list the possible Reconfix options.